### PR TITLE
Add build info to settings pane

### PR DESCRIPTION
**Pull request by vinceau**
Also basic app updating, build config variables and app version number.

### CODE REVIEW

File: build/afterSignHook.macos.js [hunk 1]
```diff
@@ -3,41 +3,27 @@ const path = require("path");
...
-const { APPLE_TEAM_PROVIDER_ID, APPLE_API_KEY, APPLE_ISSUER_ID } = process.env;
-
 module.exports = async function (params) {
   if (process.platform !== "darwin") {
     return;
   }
 
   console.log("afterSign hook triggered", params);
 
-  // Bail early if this is a fork-caused PR build, which doesn't get
```
Comment (NikhilNarayana) [inline build/afterSignHook.macos.js:15]: why remove the fail early?

File: electron-builder.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "productName": "Slippi Launcher",
-  "appId": "gg.slippi.launcher",
+  "appId": "com.github.projectslippi.slippidesktopapp",
```
Comment (NikhilNarayana) [inline electron-builder.json:3]: `slippilauncher`, i'm going to change the repo name

**Review by NikhilNarayana — CHANGES_REQUESTED** [review]

File: electron-builder.json (hunk 2 above)
Comment (vinceau) [reply electron-builder.json:3]: we can't change this app id anymore. it's what we're using to sign the macos stuff.

Comment (NikhilNarayana) [reply electron-builder.json:3]: ah rip

File: build/afterSignHook.macos.js (hunk 1 above)
Comment (vinceau) [reply build/afterSignHook.macos.js:15]: fixed in a57c7ae

**Review by NikhilNarayana — APPROVED** [review]

**Pull request merged**